src/components/cards/CardUsage.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <h1>Recent Decks using these cards</h1>
+  <div v-if="loading" class="text-center text-gray text-2xl py-8 px-4">
+    <i class="fas fa-circle-notch fa-spin"></i> Loading...
+  </div>
+  <div v-else-if="(!decks || !decks.length)" class="text-center bg-inexhaustible border border-gray-light my-4 py-8">
+    <h2>No Decks found</h2>
+    <p>
+      <button class="btn btn-blue px-4 py-2">Be the First!</button>
+    </p>
+  </div>
+  <div v-else-if="decks && decks.length">
+      <deck v-for="deck of decks" :key="deck.id" :deck="deck" :show-cards="false">
+      </deck>
+    <div v-if="limit">
+      <button class="btn px-4 py-2" @click="showAll">Show All</button>
```

Comment:
Rather than removing the limit (which actually won't "show all"; it will only return the default of 30 decks, and at best you could get back 100 decks at a time if you forced the issue), I'd prefer if you made this a link to the standard decks listing, with added support for a query-string that filters the decks by the given card stub. Granted, we'll still need proper filtration (and preferably auto-complete) support on that page eventually, but that would be a nice first step toward that.

Reply:
Done (minus the filter view)